project/plugins.sbt

```diff
@@ -5,15 +5,22 @@ addSbtPlugin("com.eed3si9n"       % "sbt-buildinfo"      % "0.9.0")
 addSbtPlugin("com.lightbend"      % "sbt-whitesource"    % "0.1.18")
 addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka"   % "0.24")
 addSbtPlugin("com.lightbend.sbt"  % "sbt-javaagent"      % "0.1.4")
-addSbtPlugin("com.thesamet"       % "sbt-protoc"         % "0.99.25")
 addSbtPlugin("com.typesafe.sbt"   % "sbt-git"            % "1.0.0")
 addSbtPlugin("com.typesafe.sbt"   % "sbt-twirl"          % "1.5.0")
 addSbtPlugin("de.heikoseeberger"  % "sbt-header"         % "5.3.1")
 addSbtPlugin("org.foundweekends"  % "sbt-bintray"        % "0.5.5")
 addSbtPlugin("com.lightbend.sbt"  % "sbt-java-formatter" % "0.4.4")
 addSbtPlugin("org.scalameta"      % "sbt-scalafmt"       % "2.3.0")
 
-addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "0.7.2")
+addSbtPlugin(
+  ("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "0.7.3")
+    // we need to exclude the sbt-protoc brought by akka-grpc (v0.99.26)
+    // because it breaks the PB.targets hack in ReflectionCodeGen
+    .exclude("com.thesamet", "sbt-protoc")
+  )
+addSbtPlugin("com.thesamet"       % "sbt-protoc"         % "0.99.25")
+  
+addSbtPlugin("com.lightbend.akka" % "sbt-akka-version-check" % "0.1")
```

Comment:
Did you mean to wire this up with CI?  Looks extra.

Reply:
That's a leftover, I added it to trace the akka versions. As you can see above, I had to explicitly add akka-discovery. 

I will add this to the CI check.